pytest_rts/utils/mappinghelper.py

```diff
@@ -0,0 +1,302 @@
+"""Module for MappingHelper class"""
+import os
+from typing import Dict, NamedTuple, Set, Tuple
+from coverage import CoverageData
+from _pytest.python import Function
+
+TestrunData = NamedTuple(
+    "TestrunData",
+    [
+        ("pytest_item", Function),
+        ("elapsed_time", float),
+        ("coverage_data", CoverageData),
+        ("found_testfiles", Set[str]),
+        ("test_function_lines", Dict[str, Dict[str, Tuple[int, int]]]),
+    ],
+)
+
+TestFunctionData = NamedTuple(
+    "TestFunctionData",
+    [
+        ("testfile_id", int),
+        ("testname", str),
+        ("start_line_number", int),
+        ("end_line_number", int),
+        ("elapsed_time", float),
+    ],
+)
+
+
+class MappingHelper:
+    """Class to handle mapping database initialization and updating"""
+
+    def __init__(self, connection):
+        """Set the connection"""
+        self.connection = connection
+
+    @property
+    def srcfiles(self):
+        """Tuple of id and path to a source code file in the mapping database"""
```

Comment:
Docstring is wrong, it's not tuple. Using mypy and typehints it is highly recommended; it would make code both more readable and machine-checkable at the same time. See:
```
@property
def srcfiles(self) -> Tuple[str, str]:
    """Return id and a path to a source code file."""
    return [blah, blah]  # mypy will complain here, as it's actually List, not Tuple
```

Reply:
(same comment about functions below)

Reply:
Ah, I completely misunderstood what this function does. Please improve docstring. (And typehint would be `-> List[Tuple[str, str]]`)